packages/block-library/src/search/index.php

```diff
@@ -131,9 +131,28 @@ function render_block_core_search( $attributes ) {
 				$button_internal_markup = wp_kses_post( $attributes['buttonText'] );
 			}
 		} else {
-			$button_classes[]       = 'has-icon';
+			$button_classes[] = 'has-icon';
+			$icon_dimensions  = '24';
+			if ( ! empty( $attributes['style']['typography']['fontSize'] ) ) {
+				$icon_dimensions = $attributes['style']['typography']['fontSize'];
+			} elseif ( ! empty( $attributes['fontSize'] ) ) {
+				$font_size_presets = wp_get_global_settings( array( 'typography', 'fontSizes' ) );
+				if ( ! empty( $font_size_presets ) ) {
+					/*
+					 * The prioritization of origins matches the order in which the block editor fetches settings.
+					 * Font sizes are sourced from either 'typography.fontSizes.custom' || 'typography.fontSizes.theme' || 'typography.fontSizes.default'.
+					 * See: `getBlockSettings()` packages/block-editor/src/store/get-block-settings.js
+					 */
+					$font_size_presets     = $font_size_presets['custom'] ?? $font_size_presets['theme'] ?? $font_size_presets['default'];
```

Comment:
Sorry for the nitpick on this one! The `block-library` PHP code gets backported with the package updates, and I think the null coalescing operator isn't yet being used in core. Should we switch to ternaries?

Reply:
>  Should we switch to ternaries?

No

>  Should we switch to ternaries?

Probably. 😄

Reply:
😂